dcos/subcommand.py

```diff
@@ -230,16 +228,19 @@ def _write_package_source(pkg):
     source_path = os.path.join(pkg_dir, 'source')
 
     with util.open_file(source_path, 'w') as source_file:
-        source_file.write(pkg.registry.source.url)
+        registry = pkg.registry()
+        if registry == "cosmos":
+            url = pkg.cosmos_url()
```

Comment:
I need to understand this better but this doesn't look right. We are suppose to write the repository url, no? Looks like we are writing cosmos' url which may include multiple repositories.

I think we need this to uniquely identify a package. A package is uniquely identified by the tuple `(repo/source url, package name, package revision)`.